### PR TITLE
Added first Lua article.

### PR DESCRIPTION
**Pull request by dknight**
I am going to write series of article about Lua, this is the first one.

### CODE REVIEW

**Review by pvalena — CHANGES_REQUESTED** [review]
Thanks! Looks good! Let's just polish it.

File: tech/languages/lua/lua_installation.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+If the output is be something like:
+
+```bash
+Lua 5.4.3  Copyright (C) 1994-2021 Lua.org, PUC-Rio
+>
+```
+
+Congratulations! Lua is already installed on your system and ready to use.
+
+<ins>Hint!</ins> To exit Lua's interpreter press <kbd>Ctrl + D</kbd>.
```
[the remark this answers is not in the document]
Comment (dknight) [reply tech/languages/lua/lua_installation.md:35]: It is compatible. Any HTML is compatible in markdown.

File: tech/languages/lua/lua_installation.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+It means that Lua is not installed yet. The simplest way to install Lua from package manager `dnf`, which comes with Fedora. In your terminal type command:
+```bash
+sudo dnf install lua
+```
+
+Congratulations! Lua interpreter is installed!
+
+## Lua syntax
+
+Lua syntax is very similar to languages Python, Ruby ad C.
```
[the remark this answers is not in the document]
Comment (dknight) [reply tech/languages/lua/lua_installation.md]: Updated

File: tech/languages/lua/lua_installation.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+```bash
+Lua 5.4.3  Copyright (C) 1994-2021 Lua.org, PUC-Rio
+>
+```
+
+Congratulations! Lua is already installed on your system and ready to use.
+
+<ins>Hint!</ins> To exit Lua's interpreter press <kbd>Ctrl + D</kbd>.
+
+If you see the message:
```
[the remark this answers is not in the document]
Comment (dknight) [reply tech/languages/lua/lua_installation.md:40]: Updated

File: tech/languages/lua/lua_installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pvalena) [reply tech/languages/lua/lua_installation.md:36]: Actually, the issue here is Jekyll, which uses sanitizer by default. Can you please check the generated page locally, if you're happy with it?
https://github.com/developer-portal/website/blob/master/DEVELOPMENT.md

File: tech/languages/lua/lua_installation.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+$ sudo dnf install lua
+```
+
+Congratulations! Lua interpreter is installed!
+
+## Lua syntax
+
+Lua syntax is very similar to languages Python, Ruby and C. To demonstrate the syntax, here is example of program to calculate factorial, from the book [Programming in Lua](https://www.lua.org/pil/1.html).
+
+```lua
```
Comment (pvalena) [inline tech/languages/lua/lua_installation.md:57]: Thanks for adding the changes, I like it!
But I'd suggest to move the example program under the `Learning lua` header instead.

What I meant for `Lua syntax`, is adding link to a command reference, like: https://www.lua.org/manual/5.4/manual.html

Comment (dknight) [reply tech/languages/lua/lua_installation.md:67]: Ok, done.

File: tech/languages/lua/lua_installation.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dknight) [reply tech/languages/lua/lua_installation.md:36]: Yes, I checked it and works file as I want. I never do changed blindly I run local server.

[the remark this answers is not in the document]
Comment (pvalena) [reply tech/languages/lua/lua_installation.md:36]: Great! Thanks for the info.

**Comment by pvalena** [comment]
LGTM, thanks!

**Pull request merged**